studygroups/api_urls.py

```diff
@@ -21,4 +21,10 @@
     url(r'^learning-circles-map/$', views.LearningCirclesMapView.as_view(), name='api_learningcircles_map'),
     url(r'^instagram-feed/$', views.InstagramFeed.as_view(), name='api_instagram_feed'),
     url(r'^teams/$', views.TeamListView.as_view(), name='api_teams'),
+    url(r'^teams/(?P<team_id>[\d]+)/$', views.TeamDetailView.as_view(), name='api_teams_detail'),
+    url(r'^teams/(?P<team_id>[\d]+)/invitation-url/create/$', views.create_team_invitation_url, name='api_teams_create_invitation_url'),
+    url(r'^teams/(?P<team_id>[\d]+)/invitation-url/delete/$', views.delete_team_invitation_url, name='api_teams_delete_invitation_url'),
+    url(r'^teams/members/$', views.TeamMembershipListView.as_view(), name='api_team_memberships'),
```

Comment:
`team_id` should be part of the URL, not part of the posted JSON. Will that break the way the `ApiHelper` works?

Reply:
@dirkcuys yeah, originally I had it as part of the URL so I could use the `user_is_team_organizer` decorator but then I'd need to either rewrite the `ApiHelper` to put variables in the base URL or rewrite the `populate_resources` function to have all the pagination logic in it... so it turned out to be easier to just pass it in the params.